river/rule/amrules.py

```diff
@@ -215,8 +298,7 @@ def _new_rule(self) -> RegRule:
             predictor = copy.deepcopy(self.pred_model)
         else:  # adaptive predictor
             predictor = AdaptiveRegressor(
-                model_predictor=copy.deepcopy(self.pred_model),
-                alpha=self.model_selector_decay,
+                model_predictor=copy.deepcopy(self.pred_model), alpha=self.alpha,
```

Comment:
You can also use the `.clone()` method that each model now has I believe.

Reply:
Good call! Just changed that in [8d249f4](https://github.com/online-ml/river/pull/594/commits/8d249f4dba1471bdf054af1a31a32a2ca927811c)